iamy/cfn_test.go

```diff
@@ -30,5 +30,17 @@ func TestCfnMangedResources(t *testing.T) {
 		if !cfn.IsManagedResource(CfnIamPolicy, "foobar-ABCDEFGH1234567") {
 			t.Fatal("names with id suffix are managed")
 		}
+
+		if cfn.IsManagedResource(CfnIamPolicy, "foobar-abcdefgh1234567") {
+			t.Fatal("names with suffix containing only lowercase letters are not managed")
```

Comment:
I find the language of this tests to be quite confusing, but realise that's not a problem created here :)

Reply:
Work for future PRs.